import{o as s,c as r,p as m,a as _,b as e,t as l,F as b,r as D,d as u,e as L,f as w,g as $,h as C,i as g,w as p,j as A}from"./vendor.bdacd7f4.js";const N=function(){const c=document.createElement("link").relList;if(c&&c.supports&&c.supports("modulepreload"))return;for(const n of document.querySelectorAll('link[rel="modulepreload"]'))a(n);new MutationObserver(n=>{for(const o of n)if(o.type==="childList")for(const i of o.addedNodes)i.tagName==="LINK"&&i.rel==="modulepreload"&&a(i)}).observe(document,{childList:!0,subtree:!0});function d(n){const o={};return n.integrity&&(o.integrity=n.integrity),n.referrerpolicy&&(o.referrerPolicy=n.referrerpolicy),n.crossorigin==="use-credentials"?o.credentials="include":n.crossorigin==="anonymous"?o.credentials="omit":o.credentials="same-origin",o}function a(n){if(n.ep)return;n.ep=!0;const o=d(n);fetch(n.href,o)}};N();var h=(t,c)=>{const d=t.__vccOpts||t;for(const[a,n]of c)d[a]=n;return d};const T={},M=t=>(m("data-v-61ba3305"),t=t(),_(),t),I={class:"pic"},H=M(()=>e("img",{src:"https://compchembio.sjtu.edu.cn/static/text/team.jpeg",alt:""},null,-1)),R=[H];function Y(t,c){return s(),r("div",I,R)}var x=h(T,[["render",Y],["__scopeId","data-v-61ba3305"]]),X="/glycosylase.jpg",k="/gp120.gif";const P={data:()=>({part1:"Computational investigations on the target-site searching and recognition mechanism by DNA glycosylase DNA glycosylase, as one member of the DNA repair machineries, plays an essential role in correcting mismatched/damaged DNA base pair by cleaving the N-glycosidic bond between the sugar and target base through the base excision repair (BER) pathways. Efficient corrections of these DNA lesions are critical for maintaining genome integrity and preventing premature aging and cancer. Therefore, an atomistic-level understanding of the dynamics of the DNA glycosylase involved in the DNA repair process can provide deep insights into the development of novel anti-cancer drugs. The target-site searching and recognition mechanism by DNA glycosylase, however, remains unknown and experimental characterization of the above process is still challenging due to the limited spatiotemporal resolutions. By employing high performance computing, combined with markov state model construction based on extensive all-atom molecular dynamics (MD) simulations, our group is currently focused on two critical human DNA glycosylases, namely thymine DNA glycosylase (TDG) and alkyladenine DNA glycosylase (AAG), regarding their target searching and substrate-recognition mechanisms. We expect to reveal the key intermediates of the above two DNA glycosylases involved in the DNA repair process. Finally, the captured intermediate states can be used as potential drug targets for future guidance to design novel anti-cancer drugs.",part2:"Computational investigations on the dynamics of the HIV envelope trimer during membrane fusion During the HIV entry, membrane fusion of the HIV and host cells is the first step and driven mainly by the glycoproteins gp120 and gp41 that form a trimer on the surface of the HIV envelope. Therefore, the gp120-gp41 complex is an important drug target to design HIV inhibitors and precursors for the HIV vaccine. However, the underlying mechanisms and atomistic-level understandings of the conformational changes of the gp120-gp41 complex induced by its receptor CD4 and co-receptors CCR5 or CXCR4 are still lack of. In current project, we are focused on investigating the dynamics of the gp120 in both its monomer and trimer state after its binding to CD4 and the refolding mechanisms of the gp41 by combining with high throughput computing, molecular dynamics simulations and Markov State Model (MSM). We expect to capture the key intermediate state of both gp120 and gp41, as well as their associated thermodynamic and kinetic properties. Finally, the above observed intermediate states can be used as potential drug targets for future structure-based anti-HIV drug discovery to design new-scaffold HIV inhibitors with high antivirus activities. In addition, our results will provide new insights for designing novel HIV vaccines that is capable of eliciting broadly neutralizing antibodies."})},y=t=>(m("data-v-76592a38"),t=t(),_(),t),J={class:"container"},B=y(()=>e("div",{class:"subtitle"},"Research",-1)),Z=y(()=>e("div",{class:"pr"}," \xA0 \xA0 \xA0The main focus of our group is to use both computational tools and experimental methods to investigate the recognition mechanisms between proteins and their specific DNA targets, i.e., how does the protein proceed to search for the target sites along the DNA chains? How does the DNA shape might affect the protein binding? How does the protein locate its specific DNA sequence among vast amounts of resembling DNA motifs etc? ",-1)),G={class:"section"},W={class:"left"},F=y(()=>e("div",{class:"right"},[e("img",{src:X,alt:"glycosylase"})],-1)),E=y(()=>e("hr",null,null,-1)),V={class:"section"},q=y(()=>e("div",{class:"left"},[e("img",{src:k,alt:"gp120"})],-1)),O={class:"right"};function z(t,c,d,a,n,o){return s(),r("div",J,[B,Z,e("div",G,[e("div",W,l(t.part1),1),F]),E,e("div",V,[q,e("div",O,l(t.part2),1)])])}var U=h(P,[["render",z],["__scopeId","data-v-76592a38"]]),Q="/dart.jpg";const K={data:()=>({people:[{name:"Jia-Qi Tian",imgSrc:"/phd/tjq.jpeg",title:"PhD",email:"tianjq@sjtu.edu.cn",duration:"2017-now"},{name:"Ling-Yan Wang",imgSrc:"/phd/wny.jpeg",title:"PhD",email:"lywang2018@sjtu.edu.cn",duration:"2018-now"},{name:"Zhi-Qiang Hao",imgSrc:"/phd/hzq.jpeg",title:"PhD",email:"haozhqiang-sjtu@sjtu.edu.cn",duration:"2020-now"},{name:"Shuang Li",imgSrc:"/master/ls.jpeg",title:"Master",email:"sli7102@sjtu.edu.cn",duration:"2017.9-2020.6"},{name:"Meng-Na Lin",imgSrc:"/master/lmn.jpeg",title:"Master",email:"linmengna@sjtu.edu.cn",duration:"2018-now"},{name:"Hong-Lin Xu",imgSrc:"/master/xhl.jpg",title:"Master",email:"honglin2019@sjtu.edu.cn",duration:"2019-now"},{name:"Kai-Yuan Song",imgSrc:"/master/sky.jpeg",title:"Master",email:"kaiyuan@sjtu.edu.cn",duration:"2020-now"},{name:"Xin-Rui Yang",imgSrc:"/master/yxr.jpg",title:"Master",email:"cloudyang61@sjtu.edu.cn",duration:"2021-now"}],pi:"Professor,  Principal investigator of Computational Chemistry lab. In 2005, obtained bachelor (Chemistry) from Inner Mongolia University. In 2005-2010, postdoctoral training at Physical Chemistry with Prof. Yun-Dong Wu in Peking University.In 2010-2013, Postdoctoral with Prof. Xuhui Huang in Chemistry Department, Hong Kong University of Science and Technology. In 2013.03-2013.09, as visiting scholar with Prof. Dong Wang in University of California, San Diego. Same year, Postdoctoral with Prof. Xuhui Huang in Chemistry Department, Hong Kong University of Science and Technology till 2014.05. Then from 2014.05 to 2015.11 as Postdoctoral with Prof. Huan-Xiang Zhou in Florida State University. In 2017, joining Shanghai Jiaotong University."})},v=t=>(m("data-v-2de3f0bf"),t=t(),_(),t),ee={class:"container"},te={class:"pi"},ne=v(()=>e("img",{src:Q,alt:""},null,-1)),ie={class:"intro"},ae=v(()=>e("div",null,[e("span",{class:"name"},"Lin-Tai Da"),e("i",{class:"piIcon"},"PI")],-1)),oe=v(()=>e("hr",null,null,-1)),se={class:"main"},re=["src"],ce={class:"title"},le={class:"name"},ue={key:0,class:"person master"},de={key:1,class:"person phd"},ge=u("email: "),he=u("In lab: "),pe=v(()=>e("div",{class:"card"},null,-1));function me(t,c,d,a,n,o){return s(),r("div",ee,[e("div",te,[ne,e("div",ie,[ae,e("span",null,l(t.pi),1)])]),oe,e("div",se,[(s(!0),r(b,null,D(t.people,i=>(s(),r("div",{class:"card",key:i.name},[e("img",{src:i.imgSrc,alt:"lmn"},null,8,re),e("div",ce,[e("div",le,[e("strong",null,l(i.name),1)]),i.title=="master"?(s(),r("div",ue,l(i.title),1)):(s(),r("div",de,l(i.title),1))]),e("div",null,[ge,e("span",null,l(i.email),1)]),e("div",null,[he,e("span",null,l(i.duration),1)])]))),128)),pe])])}var _e=h(K,[["render",me],["__scopeId","data-v-2de3f0bf"]]);const ye={data:()=>({years:[{year:2022,paper:[{title:"RQC helical hairpin in Bloom's syndrome helicase regulates DNA unwinding by dynamically intercepting nascent nucleotides.",authors:"Jianbing Ma, Chunhua Xu, Jinghua Li, Xi-Miao Hou,<strong>Lin-Tai Da</strong>, Qi Jia, Xingyuan Huang, Jin Yu, Xuguang Xi*, Ying Lu*, Ming Li.",journal:"iScience. 2022 Jan 21. 25(1), 103606.",url:"https://doi.org/10.1016/j.isci.2021.103606"}]},{year:2021,paper:[{title:"Allosteric regulation in CRISPR/Cas1-Cas2 protospacer acquisition mediated by DNA and Cas2",authors:"Chunhong Long, Liqiang Dai, Chao E, <strong>Lintai Da</strong>, Jin Yu*",journal:"Biophys J. 2021;120(15):3126-3137.",url:"https://doi.org/10.1016/j.bpj.2021.06.007"},{title:"Antigenic Peptide Loading into Major Histocompatibility Complex Class I Is Driven by the Substrate N-Terminus.",authors:"Mengna Lin, Honglin Xu, Yi Shi*, <strong>Lintai Da</strong>*.",journal:"CCS Chem. 2021, 3, 979-994",url:"https://doi.org/10.31635/ccschem.021.202000657"},{title:"Mechanism of REST/NRSF regulation of clustered protocadherin \u03B1 genes.",authors:"Yuanxiao Tang, Zhilian Jia, Honglin Xu, <strong>Lintai Da</strong>, Qiang Wu*.",journal:"Nucleic Acids Res. 2021 Apr 13:gkab248.",url:"https://doi.org/10.1093/nar/gkab248"},{title:"Atomic resolution of short-range sliding dynamics of thymine DNA glycosylase along DNA minor-groove for lesion recognition.",authors:"Jiaqi Tian, Lingyan Wang, <strong>Lintai Da</strong>*.",journal:"Nucleic Acids Res. 2021 Feb 22;49(3):1278-1293.",url:"https://doi.org/10.1093/nar/gkaa1252"}]},{year:2020,paper:[{title:"A novel neoantigen discovery approach based on chromatin high order conformation.",authors:"Yi Shi*, Mingxuan Zhang, Luming Meng*, Xianbin Su, Xueying Shang, Zehua Guo, Qingjiao Li, Mengna Lin, Xin Zou, Qing Luo, Yaoliang Yu, Yanting Wu, <strong>Lintai Da</strong>*, Tom Weidong Cai*, Guang He* & Ze-Guang Han*.",journal:"BMC Med Genomics. 2020 Aug 27;13(Suppl 6):62.",url:"https://doi.org/10.1186/s12920-020-0708-z"},{title:"DeepAntigen: A Novel Method for Neoantigen Prioritization via 3D Genome and Deep Sparse Learning.",authors:"Yi Shi*, Zehua Guo, Xianbin Su, Luming Meng*, Mingxuan Zhang, Jing Sun, Chao Wu, Minhua Zheng, Xueyin Shang, Xin Zou, Wangqiu Cheng, Yaoliang Yu, Yujia Cai, Chaoyi Zhang, Weidong Cai, <strong>Lintai Da</strong>*, Guang He*, Ze-Guang Han*. ",journal:"Bioinformatics. 2020, btaa596",url:"https://doi.org/10.1093/bioinformatics/btaa596"},{title:"Key Structural Motifs in Thymine DNA Glycosylase Responsible for Recognizing Certain DNA Bent Conformation Revealed by Atomic Simulations, Biochemical and Biophysical Research Communications. ",authors:"Shuang Li, <strong>Lin-Tai Da</strong>*.",journal:"Biochemical and Biophysical Research Communications. 2020. 526:953-959",url:"https://doi.org/10.1016/j.bbrc.2020.03.173"},{title:"Ikarugamycin inhibits pancreatic cancer cell glycolysis by targeting hexokinase 2.",authors:"Shu-Heng Jiang; Fang-Yuan Dong; <strong>Lin-Tai Da</strong>; Xiao-Mei Yang, Xiao-Xue Wang, Jing-Yi Weng, Lei Feng, Li-Li Zhu, Yan-Li Zhang, Zhi-Gang Zhang, Yong-Wei Sun, Jun Li, Min-Juan Xu*.",journal:"The FASEB Journal. 2020. 34: 3943-3955",url:"https://doi.org/10.1096/fj.201901237R"},{title:"Refolding Dynamics of gp41 from Pre-fusion to Pre-hairpin States During HIV-1 Entry.",authors:"Mengna Lin, <strong>Lin-Tai Da</strong>*.",journal:"Journal of Chemical Information and Modeling. 2020. 60: 162-174",url:"https://doi.org/10.1021/acs.jcim.9b00746"}]},{year:2019,paper:[{title:"Opening dynamics of HIV-1 gp120 upon receptor binding is dictated by a key hydrophobic core.",authors:"<strong>Lin-Tai Da</strong>*, Mengna Lin.",journal:"Physical Chemistry Chemical Physics. 2019. 21: 26003-26016",url:"https://doi.org/10.1039/C9CP04613E"},{title:"Rational Engineering of Amide Synthetase Enables Bioconversion to Diverse Xiamenmycin Derivatives.",authors:"Jingyi Weng, Xuliang Bu, Beibei He, Zhuo Cheng, Jun Xu, <strong>Lin-Tai Da</strong>*, Minjuan Xu*.",journal:"Chemical Communications. 2019. 55: 14840-14843",url:"https://doi.org/10.1039/C9CC07826F"},{title:"Characterization and Nonenzymatic Transformation of Three Types of Alkaloids from Streptomyces albogriseolus MGR072 and Discovery of Inhibitors of Indoleamine 2,3-Dioxygenase.",authors:"Du Gao, Ting Zhou, <strong>Lin-Tai Da</strong>, Torsten Bruhn, Li-Li Guo, Yu-Hang Chen, Jun Xu, Min-Juan Xu*.",journal:"Organic letters. 2019. 21(21):8577-8581",url:"https://doi.org/10.1021/acs.orglett.9b03149"},{title:"Regulatory Role of One Critical Catalytic Loop of Polypeptide N-Acetyl-Galactosaminyltransferase-2 in Substrate Binding and Catalysis during Mucin-Type O-Glycosylation.",authors:"Jiaqi Tian, Feng Liu, Zhijue Xu, Jingjing Shi, Tao Liang, Yan Zhang*, <strong>Lin-Tai Da</strong>*.",journal:"ACS Catalysis 2019. 9(11):10536-10550",url:"https://pubs.acs.org/doi/full/10.1021/acscatal.9b03782"},{title:"Inhibition of polypeptide N-acetyl-\u03B1-galactosaminyltransferases is an underlying mechanism of dietary polyphenols preventing colorectal tumorigenesis.",authors:"Feng Liu, Yalu Cui, Fang Yang, Zhijue Xu, <strong>Lin-Tai Da</strong>, Yan Zhang*.",journal:"Bioorganic & Medicinal Chemistry. 2019. 27(15): 3372-3382",url:"https://doi.org/10.1016/j.bmc.2019.06.020"},{title:"pH-Induced Misfolding Mechanism of Prion Protein: Insights from Microsecond-Accelerated Molecular Dynamics Simulations.",authors:"Shuangyan Zhou, Danfeng Shi, Xuewei Liu, Xiaojun Yao, <strong>Lin-Tai Da</strong>*, Huanxiang Liu*. ",journal:"ACS Chem. Neurosci. 2019. 10: 2718-2729",url:"https://pubs.acs.org/doi/10.1021/acschemneuro.8b00582"},{title:"Determining selection free energetics from nucleotide pre-insertion to insertion in viral T7 RNA polymerase transcription fidelity control.",authors:"Chunhong Long, Chao E, <strong>Lin-Tai Da</strong>, Jin Yu*. ",journal:"Nucleic Acids Res. 2019. 47(9): 4721-4735",url:"https://doi.org/10.1093/nar/gkz213"},{title:"Understanding the molecular mechanism of umami recognition by T1R1-T1R3 using molecular dynamics simulations. ",authors:"Hai Liu, <strong>Lin-Tai Da</strong>*, Yuan Liu*. ",journal:"Biochemical and Biophysical Research Communications. 2019. 514(3): 967-973",url:"https://doi.org/10.1016/j.bbrc.2019.05.066"},{title:"A viral T7 RNA polymerase ratcheting along DNA with fidelity control.",authors:"Chunhong Long, Chao E, <strong>Lin-Tai Da</strong>, Jin Yu*. ",journal:"Computational and Structural Biotechnology Journal. 2019. 17: 638-644",url:"https://doi.org/10.1016/j.csbj.2019.05.001"}]},{year:2018,paper:[{title:"Base-flipping dynamics from an intrahelical to an extrahelical state exerted by thymine DNA glycosylase during DNA repair process.",authors:"<strong>Lin-Tai Da</strong>*, Jin Yu.",journal:"Nucleic Acids Res. 2018 46(11): 5410-5425",url:"https://academic.oup.com/nar/article/46/11/5410/4995696"},{title:"Dynamics of the excised base release in thymine DNA glycosylase during DNA repair process. ",authors:"<strong>Lin-Tai Da</strong>*, Yi Shi, Guodong Ning and Jin Yu*.",journal:"Nucleic Acids Res. 2018 46(2): 568-581",url:"https://academic.oup.com/nar/article/46/2/568/4739373"}]},{year:2017,paper:[{title:"T7 RNA polymerase translocation is facilitated by a helix opening on the fingers domain that may also prevent backtracking.",authors:"<strong>Lin-Tai Da</strong>*, Chao E, Yao Shuai, Shaogui Wu, Xiao-Dong Su, Jin Yu*. ",journal:"Nucleic Acids Res. 2017 45(13): 7909-7921",url:"https://academic.oup.com/nar/article/45/13/7909/3859021"}]}],url:["http://www.nature.com/articles/ncomms11244","http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004624","http://link.springer.com/chapter/10.1007%2F978-3-319-02970-2_2","http://www.ncbi.nlm.nih.gov/pubmed/25475208","http://www.ncbi.nlm.nih.gov/pubmed/25418098","http://www.sciencedirect.com/science/article/pii/S1568786414000950","http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0097975","http://www.pnas.org/content/111/21/7665.short","http://www.worldscientific.com/doi/abs/10.1142/S0219633613410058","http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003020","http://pubs.acs.org/doi/abs/10.1021/ja210656k","http://pubs.acs.org/doi/abs/10.1021/ja208720a","http://pubs.acs.org/doi/abs/10.1021/ci1003448","http://onlinelibrary.wiley.com/doi/10.1002/prot.23005/references","http://pubs.acs.org/doi/abs/10.1021/jp9081239"]})},j=t=>(m("data-v-b2bd417a"),t=t(),_(),t),fe={class:"bg-gradient_solid"},ve={class:"container"},be=j(()=>e("div",{class:"section-header"},[e("h2",null,"Publications"),e("hr")],-1)),De={class:"steps"},Ce={class:"content"},je={class:"paper"},Se=["href"],Le=["innerHTML"],we={class:"journal"},$e=j(()=>e("i",{class:"step-line"},null,-1)),Ae={class:"date"};function Ne(t,c,d,a,n,o){return s(),r("div",fe,[e("div",ve,[be,e("div",De,[(s(!0),r(b,null,D(t.years,i=>(s(),r("div",{class:"steps-container",key:i.year},[e("div",Ce,[(s(!0),r(b,null,D(i.paper,f=>(s(),r("div",je,[e("a",{class:"title",href:f.url,target:"_blank"},l(f.title),9,Se),e("div",{class:"authors",innerHTML:f.authors},null,8,Le),e("div",we,l(f.journal),1)]))),256))]),$e,e("div",Ae,l(i.year),1)]))),128))])])])}var Te=h(ye,[["render",Ne],["__scopeId","data-v-b2bd417a"]]);const Me={};function Ie(t,c){return null}var He=h(Me,[["render",Ie]]),Re="/map.png";const Ye={},xe={class:"container"},Xe=L('<div class="contact" data-v-31065011><div class="address" data-v-31065011><h2 data-v-31065011>Shanghai Center for Systems Biomedicine</h2><div class="detail" data-v-31065011><p data-v-31065011> Shanghai Jiao Tong University<br data-v-31065011> RM B300 SCSB BLDG,<br data-v-31065011> No.800, Dong Chuan Rd. Shanghai, 200240 </p><span class="email" data-v-31065011>Email\uFF1A darlt@sjtu.edu.cn</span></div></div><div class="map" data-v-31065011><img src="'+Re+'" data-v-31065011></div></div>',1),ke=[Xe];function Pe(t,c){return s(),r("div",xe,ke)}var Je=h(Ye,[["render",Pe],["__scopeId","data-v-31065011"]]);const Be=[{path:"/",component:x},{path:"/research",component:U},{path:"/publication",component:Te},{path:"/member",component:_e},{path:"/opening",component:He},{path:"/contact",component:Je}],Ze=w({history:$(),routes:Be});const Ge=t=>(m("data-v-11cdce93"),t=t(),_(),t),We=Ge(()=>e("div",{class:"logo"},[e("p",{id:"h1"},[e("img",{src:"https://compchembio.sjtu.edu.cn/static/text/sss.png",alt:""}),u(" \xA0\xA0\xA0Computational Chemistry Group ")])],-1)),Fe={class:"nav"},Ee={class:"menu"},Ve=u("Home"),qe=u("Research"),Oe=u("Members"),ze=u("Publication"),Ue=u("Opening"),Qe=u("Contact"),Ke={setup(t){return(c,d)=>{const a=C("router-link"),n=C("router-view");return s(),r("div",null,[We,e("div",Fe,[e("ul",Ee,[e("li",null,[g(a,{to:"/"},{default:p(()=>[Ve]),_:1})]),e("li",null,[g(a,{to:"/research"},{default:p(()=>[qe]),_:1})]),e("li",null,[g(a,{to:"/member"},{default:p(()=>[Oe]),_:1})]),e("li",null,[g(a,{to:"/publication"},{default:p(()=>[ze]),_:1})]),e("li",null,[g(a,{to:"/opening"},{default:p(()=>[Ue]),_:1})]),e("li",null,[g(a,{to:"/contact"},{default:p(()=>[Qe]),_:1})])])]),g(n)])}}};var et=h(Ke,[["__scopeId","data-v-11cdce93"]]);const S=A(et);S.use(Ze);S.mount("#app");
